src/Assets/Assets.php

```diff
@@ -109,22 +109,21 @@ public static function asset( string $slug, string $file, string $version = null
 	 *
 	 * @param Asset $asset Register an asset.
 	 *
-	 * @return Asset|false The registered object or false on error.
+	 * @return Asset|VendorAsset The registered object or false on error.
```

Comment:
```suggestion
	 * @return Asset|VendorAsset|false The registered object or false on error.
```

Reply:
It's impossible for this method to return `false`. That's why I removed it from the docblock.